docs/notes/previous_versions/v2.14.2.rst

```diff
@@ -0,0 +1,39 @@
+`Version 2.14.2 (latest) <https://fast-dds.docs.eprosima.com/en/v2.14.2/index.html>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important::
+
+    Fast DDS v2.14 will be the last minor version of Fast DDS v2, the next major release will be Fast DDS
+    v3.0.0, stay tuned!
+
+This release includes the following **features** in an ABI compatible manner:
+
+#. Set :ref:`DataSharing Qos policy<datasharingqospolicy>` in transmitted ``WriterProxyData`` and ``ReaderProxyData``.
+#. New :ref:`max_message_size<use-case-large-data-options>` property to limit the output datagrams size.
+#. Add XML configuration for :ref:`flow-controllers`.
+
+This release includes the following **improvements**:
+
+#. Repository & CI improvements.
+#. Do not require `PYTHON_VERSION` to be defined. in .bat files.
+#. Use `%*` instead of loop in `.bat` scripts.
```

Comment:
Otherwise, it is displayed as italic style
```suggestion
#. Do not require ``PYTHON_VERSION`` to be defined. in .bat files.
#. Use ``%*`` instead of loop in ``.bat`` scripts.
```